packages/frontend/src/components/SimpleChart/index.tsx

```diff
@@ -105,13 +114,22 @@ const SimpleChart: FC<SimpleChartProps> = memo((props) => {
         [onSeriesContextMenu, eChartsOptions],
     );
 
+    const onLegendChange = useCallback(
+        (params: LegendClickEvent) => {
+            setSelectedLegends(params.selected);
+        },
+        [],
+    );
+
     const onEvents = useMemo(
         () => ({
             contextmenu: onChartContextMenu,
             click: onChartContextMenu,
+            legendselectchanged: onLegendChange,
```

Comment:
I do'nt see any reference to this `legendselectchanged`...

Reply:
legendselectchanged gives a list of all legends that are currently available and their selection status - true/false